### PR TITLE
add SAN CSR vectors

### PR DESCRIPTION
**Pull request by reaperhulk**

### CODE REVIEW

File: docs/development/test-vectors.rst [hunk 1]
```diff
@@ -146,6 +146,9 @@ Custom X.509 Request Vectors
   SHA256 generated using OpenSSL.
 * ``ec_sha256.pem`` - Contains a certificate request using EC (``secp384r1``)
   and SHA256 generated using OpenSSL.
+* ``san_rsa_sha256.pem`` and ``san_rsa_sha256.der`` - Contain a certificate
```
Comment (alex) [inline docs/development/test-vectors.rst]: These both appear to be signed with `sha1WithRSAEncryption`, not `sha256`.

Comment (reaperhulk) [inline docs/development/test-vectors.rst]: Oops, updated PR to name the files properly and put the right information in the docs.

File: docs/development/test-vectors.rst [hunk 2]
```diff
@@ -136,16 +136,19 @@ Custom X.509 Vectors
...
+  generated using OpenSSL.
+* ``rsa_sha1.pem`` - Contains a certificate request using 2048 bit RSA and
+  SHA1 generated using OpenSSL.
+* ``rsa_sha256.pem`` - Contains a certificate request using 2048 bit RSA and
+  SHA256 generated using OpenSSL.
+* ``ec_sha256.pem`` - Contains a certificate request using EC (``secp384r1``)
+  and SHA256 generated using OpenSSL.
+* ``san_rsa_sha1.pem`` and ``san_rsa_sha1.der`` - Contain a certificate
+  request using RSA and SHA1 with a subject alternative name extension
+  generated using OpenSSL.
```
Comment (alex) [inline docs/development/test-vectors.rst]: This merge was not resolved correctly.

**Comment by reaperhulk** [comment]
I've resolved the bad rebase.

**Comment by alex** [comment]
lgtm

**Pull request merged**